guides/v2.0/contributor-guide/contributing.md

```diff
@@ -214,10 +214,15 @@ Refer to the following table for a description of each label. These labels are a
       <tr>
          <td><img src="{{ site.baseurl }}common/images/github_Cannot_Reproduce.png" alt="Can Not Reproduce"/></td>
          <td>The Community Engineering Team can not reproduced the issue following steps to reproduce.</td>
-      </tr>  
+      </tr>
       <tr>
          <td><img src="{{ site.baseurl }}common/images/github_non-issue.png" alt="Non Issue"/></td>
          <td>A described behavior in the issue description is valid and shouldn't be changed in Magento code base.</td>
       </tr>
    </tbody>
+
+<h2 id="triage">Help triage issues <a href="https://www.codetriage.com/magento/magento2"><img src="https://www.codetriage.com/magento/magento2/badges/users.svg" /></a></h2>
+
+In addition to contributing code, you can help to triage issues. This can include reproducing bug reports, or asking for vital information such as version numbers or reproduction instructions. If you would like to start triaging issues, one easy way to get started is to [subscribe to Magento on CodeTriage](https://www.codetriage.com/magento/magento2).
```

Comment:
1. Move comma and revise wording: _This can include reproducing bug reports or asking for vital information**,** such as **affected versions** or **instructions to reproduce bugs**._
2. Please append `{:target="_blank"}` to the link to open in a new browser tab.